src/components/Clock/Clock.tsx

```diff
@@ -29,9 +32,25 @@ const getDegreeByDate = (m: Moment, hande: ClockHand): number => {
 };
 
 export default function Clock(props: ClockProps) {
+  const [modalOpen, setModalOpen] = useState(false);
+
+  const handleButtonClick = () => {
+    setModalOpen(false);
+  };
+
   return (
     <>
       <div className={styles.clockwrapper}>
+        <button className={styles.buttonclock} onClick={() => setModalOpen(true)}>
+          {props.nameevent}
+        </button>
+        {modalOpen && (
+          <Modal
+            onSave={handleButtonClick}
+            onCansel={handleButtonClick}
```

Comment:
```suggestion
            onCancel={handleButtonClick}
```

Reply:
I can't understand the problem

Reply:
a typo. Should be can **c** el instead of can **s** el

Reply:
fixed it